spec/signup_error_handler_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'signup_error_handler'
+
+describe SignupErrorHandler do
+  context "when passed an error during sign-up validation" do
+    let(:errors) {
+      [['Username is already taken'],
+       ['Username must not be blank'],
+       ['Email is already taken'],
+       ['Email has an invalid format'],
+       ['Email must not be blank'],
+       ['Password does not match the confirmation']]
+     }
```

Comment:
Lint/BlockAlignment: `end` at 12, 5 is not aligned with `let(:errors) {` at 5, 4.